spring-boot-test-autoconfigure/src/test/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+# Use random port for embedded Mongod.
+spring.data.mongodb.port = 0
```

Comment:
Would you mind pointing me into the right direction? Would `@TestPropertySource` a good tool?